trace-C19/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,92 @@
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  background-color: #eeda73 !important;
+  text-align: center;
+  color: black;
+}
+
+.title {
+  color: black;
+  font-style: bold;
+  font-size: 400%;
+  text-align: center;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Lato", sans-serif;
+}
+
+.description {
+  font-size: 130%;
+  text-align: center;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Muli", sans-serif;
+  font-family: "Lato", sans-serif;
+}
+
+.point {
+  font-size: 50%;
+}
+
+.question {
+  font-size: 150%;
+  text-align: center;
+  font-family: "Lato", sans-serif;
+}
+
+.mission {
+  text-align: center;
+  font-family: "Lato", sans-serif;
+}
+
+.image {
+  text-align: center;
+  background-attachment: fixed;
+}
+.headline {
+  top: 10vh;
+}
+
+.btn-primary {
+  background-color: black;
+  color: white;
+}
+
+nav {
+  background-color: black;
+  color: white;
+}
+
+.navbar {
+  background-color: #2e3587 !important;
+}
+
+.nav-item {
+  color: #white !important;
+  font-family: "Noto Sans", sans-serif;
+  font-family: "Lato", sans-serif;
```

Comment:
Same here

Reply:
The reason is that it is needed to make the font I want unique.

Reply:
Oh Okay. I thought that twas the result of the branch being merged with unresolved conflicts. 
Because double font families in one class are highlighted as errors in my cloud shell IDE. 

Since you said you meant to have both font-families, I was thinking maybe it would be nice to have different classes each hosting a different font family.

```
.fon1{
  font-family: "Noto Sans", sans-serif;
}

.font2{
   font-family: "Lato", sans-serif;
}
```
Then reference these classes in your html based on your need.

Just a suggestion :)